group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -619,6 +622,22 @@ ConsumerGroup getOrMaybeCreateConsumerGroup(
         }
     }
 
+    /**
+     * Gets a consumer group.
+     *
+     * @param groupId           The group id.
+     *
+     * @return A ConsumerGroup.
+     * @throws GroupIdNotFoundException if the group does not exist or the group is not a consumer group.
+     *
+     * Package private for testing.
+     */
+    ConsumerGroup getConsumerGroup(
```

Comment:
I just noticed that we also have `consumerGroup(String, long)` as a method. I wonder whether we should also name this one `consumerGroup` too and call the one that I mentioned. It would also make sense to regroup them. What do you think?